### ISSUE TITLE
Wrong `Fixes` are constructed when `Sequence type` is involved

### ISSUE THREAD

**Issue by suhasgumma** [kubescape/kubescape#993]
Currently, Kubescape is not accounting for the `Sequences` when constructing `fixes` for `auto-fixing files` or displaying in `Assistant Remediation`.

## Example while auto-fixing files: 
* One of the basic fixes is to `drop "NET_RAW" capability`. 
* Fix Constructed: `spec.containers[0].securityContext.capabilities.drop |= NET_RAW`. 
* Right fix : `spec.containers[0].securityContext.capabilities.drop +=  ["NET_RAW"]`
```
// Fix Applied with the constructed path
capabilities:
     drop: NET_RAW

// Right Fix:
capabilities:
     drop: ["NET_RAW"]

// OR
capabilities:
     drop:
     - NET_RAW
```

## TLDR
Fixes should be constructed differently for `mapping` and `sequence` types.

## Possible Solution
* Add another field `PathType` to `FixPath` in `armosec/armoapi-go`
https://github.com/armosec/armoapi-go/blob/392cab84a55ee919cd1559da112c00b00af11d06/armotypes/posturetypes.go#L287

* The `PathType` field takes either `Mapping` or `Sequence` as values. The fixes can be constructed accordingly knowing the type of Path.

**Comment by dwertent** [comment]
cc: @yuleib

**Comment by YiscahLevySilas1** [comment]
fixed in [regolibrary PR #230](https://github.com/kubescape/regolibrary/pull/230)

**Comment by dwertent** [comment]
I think it is better we close the issue only after the fix is published :)
